src/WebJobs.Script/Binding/ExtensionBinding.cs

```diff
@@ -19,7 +19,7 @@ namespace Microsoft.Azure.WebJobs.Script.Binding
     /// Wrapper used to adapt a <see cref="ScriptBinding"/> to the binding pipeline.
     /// </summary>
     [CLSCompliant(false)]
-    public class ExtensionBinding : FunctionBinding
+    public class ExtensionBinding : FunctionBinding, IResultProcessingBinding
```

Comment:
All this IResultProcessor stuff is a hack that we've been hoping to remove, but I see it's use still grows... What are you using it for exactly? It was kind of a hack, only used for http binding. @fabiocav